src/kaleidoscope/Kaleidoscope.h

```diff
@@ -48,9 +51,9 @@ extern HARDWARE_IMPLEMENTATION KeyboardHardware;
 #define KEYMAP(...) { PER_KEY_DATA(XXX, __VA_ARGS__) }
 #endif
 
-#define ROWS (KeyboardHardware.matrix_rows)
-#define COLS (KeyboardHardware.matrix_columns)
-#define LED_COUNT (KeyboardHardware.led_count)
+#define ROWS (kaleidoscope_internal::device.matrix_rows)
+#define COLS (kaleidoscope_internal::device.matrix_columns)
+#define LED_COUNT (kaleidoscope_internal::device.led_count)
```

Comment:
Do these three macros get called anywhere anymore after the refactoring? They may want deprecation warnings added.

Reply:
It looks like there are...just a couple of usages left. Those should die.

Reply:
I can kill them, and I can deprecate them too. Should I deprecate, or just kill them with fire?

Reply:
Deprecate it is.